src/optimagic/optimization/algo_options.py

```diff
@@ -108,6 +108,25 @@
 
 """
 
+CONVERGENCE_TARGET_CRITERION = None
+"""float or None: Stop when the criterion value is better than or equal to
+    this target. The definition of "better" depends on the optimization direction.
+
+    - Minimization: criterion <= target
+    - Maximization: criterion >= target
+
+    Used in population-based algorithms like genetic algorithms.
+    To disable, set to None.
+"""
+
+CONVERGENCE_SATURATE_GENERATIONS = None
```

Comment:
Let's rename to `convergence_generations_noimprove` to align it better with [this](https://github.com/optimagic-dev/optimagic/blob/1f95f25223933a693ce9ca7e25cd73459ae93aed/src/optimagic/optimizers/nevergrad_optimizers.py#L153)